scipy/stats/tests/test_distributions.py

```diff
@@ -3258,6 +3258,26 @@ def test_ncx2_tails_pdf():
     assert_(np.isneginf(logval).all())
 
 
+@pytest.mark.parametrize('method, expected', [
+    ('cdf', np.array([1.72115630e-04, 2.74102033e-05])),
+    ('pdf', np.array([0.00078975, 0.00013013])),
+    ('logpdf', np.array([-7.14378973, -8.94700192]))
+])
+def test_ncx2_zero_nc(method, expected):
+    # gh-5441
+    # ncx2 with nc=0 is identical to chi2
+    result = getattr(stats.ncx2, method)(1, nc=[0, 4], df=10)
+    assert_allclose(result, expected, rtol=1e-3, atol=0)
```

Comment:
Why this combinbation of tolerances? 
Assuming that the numbers come from chi2 distribution for nc=0 and ncx2 for nc != 0, the tolerances could/should be made way tighter, close 1e-15 or some such.